spot-controller/src/setup.js

```diff
@@ -86,6 +116,14 @@ export default class Setup extends React.Component {
      * @returns {void}
      */
     _onSubmitEnteredUrl() {
-        this.props.onSubmitEnteredUrl(this.state.enteredRemoteControlUrl);
+        try {
+            const urlParts = url.parse(this.state.enteredRemoteControlUrl);
+
+            if (/localhost|jitsi.net|8x8.vc/.exec(urlParts.hostname)) {
```

Comment:
This is the whitelist part. This feels pretty limiting but maybe it's okay for now, meaning the next n-weeks.

Reply:
Maybe it'd be better to have a lock for this feature? Not sure how we'd go about setting the lock. But I still think this is okay for now, as long as we decide what to do and do it before the next release.